src/App.js

```diff
@@ -58,22 +60,40 @@ class App extends React.Component {
   openModal(lastClicked) {
     var modal = document.getElementById("info-modal");
     var span = document.getElementsByClassName("sc-modal-close")[0];
+    var currentScroll = window.scrollY;
+
     modal.style.display = "block";
     modal.scrollTop = 0;
 
+    // When the modal is shown, we want a fixed body
```

Comment:
Do these have a negative effect on the embedding experience? I'm slightly fearful of merging these in, in case we do decide to go the embed route. 

(though, seems like this may be fine embedding anyways?)

Reply:
It shouldn't have a negative effect! (although there are a ton of other CSS issues with the embed option)